test/jasmine/tests/mapbox_test.js

```diff
@@ -245,6 +245,32 @@ describe('mapbox credentials', function() {
             done();
         });
     });
+
+    it('should bypass access token in mapbox layout options when config points to an Atlas server', function(done) {
+        var cnt = 0;
+        var msg = [
+            'An API access token is required to use Mapbox GL.',
+            'See https://www.mapbox.com/developers/api/#access-tokens'
+        ].join(' ');
+
+        Plotly.plot(gd, [{
```

Comment:
Hmm, because there's an error thrown we can't catch yr `''` return from `plot`, right?